src/main/java/org/mule/extension/db/internal/domain/executor/BulkUpdateExecutor.java

```diff
@@ -98,4 +89,30 @@ public Object execute(DbConnection connection, Query query, List<List<QueryParam
       statement.close();
     }
   }
+
+  private void logBulkUpdateErrorInfo(int[] updateCounts, int batchCount) {
```

Comment:
Method could be reused to log results (even when finished ok)

Reply:
I couldn't reuse it after all, because I needed the BulkUpdateException object to obtain that array. I added an info log, nevertheless, to leave a record of how many statements were executed.